docs-ref-services/activedirectory.md

```diff
@@ -34,7 +34,7 @@ npm install adal-node
 
```

Comment:
This package doesn't need to be updated?

Reply:
yes, Ramya has mentioned the reason above